### ISSUE TITLE
Inherit unexpected behavior

### ISSUE THREAD

**Issue by ifalldev** [aldeed/meteor-template-extension#54]
hi, I'm using the `juliancwirko:s-image-box` package and _replacing its main template_ (sImageBox).

But i need the `sImageBox` template _to inherit the helpers_ from my new template [the one which is replacing (newImageBox)] and _to inherit the helper from another template_ (contentCard).

Ok, but doing this: 
```
Template.imageBoxNew.replaces('sImageBox');
Template.sImageBox.inheritsHelpersFrom('newImageBox');
Template.sImageBox.inheritsHelpersFrom('contentCard');
```

or this:
```
Template.newImageBox.inheritsHelpersFrom('contentCard');
Template.imageBoxNew.replaces('sImageBox');
Template.sImageBox.inheritsHelpersFrom('newImageBox');
```
on `newImageBox.js` file, the result is that my `sImageBox` _only inherits helpers from my `newImageBox` template_.

Then I put the command:
`Template.sImageBox.inheritsHelpersFrom('contentCard');`
on my `contentCard.js` file and it worked well.

I'm just reporting the case because I didn't find it on docs.

**Comment by aldeed** [comment]
It's a bit hard to follow the example code, but this seems to be working correct. Calling inheritsHelpersFrom multiple times should be fine as long as all the templates you're inheriting from load before your code runs.
